msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -135,9 +135,13 @@ If your app makes use of [Content Security Policy](https://developer.mozilla.org
     |Teams web |5e3ce6c0-2b1f-4285-8d4b-75ee78787346 |
     |Office web  |4765445b-32c6-49b0-83e6-1d93765276ca|
     |Office desktop  | 0ec893e0-5785-4de6-99da-4ed124e5296c |
+    |Office mobile  | d3590ed6-52b3-4102-aeff-aad2292ab01c |
     |Outlook desktop, mobile | d3590ed6-52b3-4102-aeff-aad2292ab01c |
     |Outlook web | bc59ab01-8403-45c6-8796-ac3ef710b3e3|
 
+    > [!NOTE]
+    > Microsoft 365 client applications might share Client IDs.
```

Comment:
Instead of "might" which sounds indefinite, could we say more affirmatively:

Some Microsoft 365 client applications share Client IDs.